sdk/eventhub/azure-eventhubs/azure/eventhub/async_ops/__init__.py

```diff
@@ -204,7 +204,8 @@ def _create_auth(self, username=None, password=None):
 
     def add_async_receiver(
             self, consumer_group, partition, offset=None, prefetch=300,
-            operation=None, keep_alive=30, auto_reconnect=True, loop=None):
+            operation=None, keep_alive=30, auto_reconnect=True, loop=None,
+            idle_timeout=None):
```

Comment:
Does the docstring need to be updated for `idle_timeout`?

Reply:
Probably!  I have to ask however, I notice there are a few othere params missing docstrings.  Did we have a reason for this? (I could think of e.g. since they're somewhat esoteric settings, but even then docs seem... pretty harmless)  I can backfill at least some if not.